packages/utils/src/error.ts

```diff
@@ -57,6 +57,8 @@ export const ERROR_TYPE = enumify({
   UNSUPPORTED_NOTIFICATION: "UNSUPPORTED_NOTIFICATION",
   UNSUPPORTED_SIGNAL: "UNSUPPORTED_SIGNAL",
   USER_DISCONNECTED: "USER_DISCONNECTED",
+  // 8000 (Not Initialized)
+  NOT_INITIALIZED: "NOT_INITIALIZED",
```

Comment:
Doesn't this belong in the internal section i.e. `1xxx`? Why are we creating a new category for this?